### DISCUSSION TITLE
How to migrate from Astro v2 to v3 for existing Cactus project?

### DISCUSSION THREAD

**Discussion by ajitzero** [chrismwilliams/astro-theme-cactus#147] [Q&A] [answered]
I generated my blog from Cactus's v2 version a few months ago and want to try upgrading it.

I am comfortable enough with Astro to migrate it manually, but I was wondering if there was anything Cactus-specific to keep in mind or if I should just use the official [Astro migration guide](https://docs.astro.build/en/guides/upgrade-to/v3/).

**Comment by chrismwilliams** [comment, marked as answer]
Hi @ajitzero, hopefully your already sorted but if not, definitely use the migration guide as the docs provided by Astro are usually really good

**Reply by ajitzero** [reply]
Yes, I followed the Astro guide to get an idea of the high-level changes, but I ended up cloning this repo again and compared all the files manually 😓 It took less time than I thought and works fine for now.
